src/com/google/javascript/jscomp/AbstractCompiler.java

```diff
@@ -477,7 +477,7 @@ public boolean isNormalizedObfuscated() {
    */
   public abstract Node getRoot();
 
-  abstract CompilerOptions getOptions();
+  public abstract CompilerOptions getOptions();
```

Comment:
I'd rather this stay package-private. I think it's a better design to refactor the coverage passes to take in the specific option values they need.

But I'm fine with making these other methods public.

Reply:
Updated.